javascript/app.js

```diff
@@ -2,15 +2,215 @@ const menuBars = document.getElementById('menu-bars');
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    urlMobile: 'images/work_2.jpg',
+    urlDesktop: 'images/work_2_lg.png',
+    urlModal: '/images/work_modal.png',
+    alt: 'work_multi-post_stories_daily_personalized_reads',
+    live: 'https://dennis-every.github.io/',
+    source: 'https://github.com/dennis-every/portfolio',
+  },
+  {
+    id: 2,
+    title: 'Tonic',
+    experiences: ['Canopy', 'Back End Dev', '2015'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    urlMobile: 'images/work_3.jpg',
+    urlDesktop: 'images/work_3_lg.png',
+    urlModal: '/images/work_modal.png',
+    alt: 'work_tonic_daily_personalized_reads',
+    live: 'https://dennis-every.github.io/',
+    source: 'https://github.com/dennis-every/portfolio',
+  },
+  {
+    id: 3,
+    title: 'Multi-Post Stories',
+    experiences: ['Canopy', 'Back End Dev', '2015'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    urlMobile: 'images/work_4.jpg',
+    urlDesktop: 'images/work_4_lg.png',
+    urlModal: '/images/work_modal.png',
+    alt: 'work_multi-post_stories_daily_personalized_reads',
+    live: 'https://dennis-every.github.io/',
+    source: 'https://github.com/dennis-every/portfolio',
+  },
+];
+
+const worksLoadHandler = () => {
+  const works = document.getElementById('works');
+  worksArray.forEach((work) => {
+    const workCard = document.createElement('li');
+    workCard.classList.add('work-card');
+    workCard.innerHTML = `
+      <article class='work-card__article'>
+        <div class='work-card__header'>
+          <picture>
+            <source srcset=${work.urlDesktop} media='(min-width: 768px)' />
+            <img
+              src=${work.urlMobile}
+              alt=${work.alt}
+            />
+          </picture>
+        </div>
+        <div class='work-card__body'>
+          <div class='elements'>
+            <header class='work-card__title'>${work.title}</header>
+            <div>
+              <ul class='work-card__experience'>
```

Comment:
As requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/209001206-5d022bd2-75fe-4891-8dc1-e0915b5d9106.png)

Kindly remember that you need to follow the original design 😉. Currently, all of your work cards are being displayed in the same way (image on the left & text on the right). Please notice that, in the original design, the project cards alternate between an image on the left & image on the right:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/209001374-bf6c242c-8366-4b7d-8439-7dd017083e8d.png)  | ![image](https://user-images.githubusercontent.com/42799579/209001448-a830e616-2b12-4b02-8abf-d3ede4d80a60.png)  |

> Keep in mind that we need to check your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.